CONTRIBUTING.md

```diff
@@ -103,63 +111,68 @@ github_changelog_generator liferay/alloy-editor -t $GITHUB_ACCESS_TOKEN
 
-## Hot fix
+```
+git tag $VERSION -m $VERSION
+npm publish --dry-run # Final sanity check.
+npm publish
+git push --follow-tags upstream
+```
 
-1. Create a feature branch from `stable` (assuming hot fix has already been
-merged)
+## 10. Sync `stable` and `master`
 
 ```
 git checkout stable
-git pull upstream stable
-git checkout -b feature/fix_foo
+git merge master # generally, will be a fast-forward merge
+git push upstream stable
 ```
 
-2. Send a fix PR to `stable`
+## 11. Update GitHub release information using the pushed vX.X.X tag and the appropriate portion of CHANGELOG.md
 
-3. Follow steps 3-11 of a scheduled release
+See [the Releases listing](https://github.com/liferay/alloy-editor/releases) on GitHub.
 
 ## Updating CKEditor.
 
-1. Open your favourite browser and navigate to https://ckeditor.com/cke4/builder.
+### 1. Open your favourite browser and navigate to https://ckeditor.com/cke4/builder.
 
-2. On this page you should see a button labelled "Upload build-config.js", *click it*:
+### 2. On this page you should see a button labelled "Upload build-config.js", *click it*:
 
-	This will open a file dialog letting you choose CKEditor's build configuration file:
+This will open a file dialog letting you choose CKEditor's build configuration file:
 
-	This file is located in `lib/ckeditor/build-config.js`, select it to upload it.
+This file is located in `lib/ckeditor/build-config.js`, select it to upload it.
 
-4. Optionally if you want to add more plugins, scroll down and you should see two panels:
+### 4. Optionally if you want to add more plugins, scroll down and you should see two panels:
 
-	On the left you'll see the list of selected plugins (automatically detected by parsing the `build-config.js` file uploaded previously) and on the right a list of the available plugins. Click on the plugins you wish to add and on the left arrow button ("<"), this will add the new plugins into the "selected plugins" panel.
+On the left you'll see the list of selected plugins (automatically detected by parsing the `build-config.js` file uploaded previously) and on the right a list of the available plugins. Click on the plugins you wish to add and on the left arrow button ("<"), this will add the new plugins into the "selected plugins" panel.
 
-5. Finally, scroll to the bottom of the page, agree to the terms of use and click the "Download CKEditor" button.
+### 5. Finally, scroll to the bottom of the page, agree to the terms of use and click the "Download CKEditor" button.
 
-6. Once the file downloaded, unzip it and *replace* the `lib/ckeditor` directory with the new directory you unzipped.
+### 6. Once the file downloaded, unzip it and *replace* the `lib/ckeditor` directory with the new directory you unzipped.
 
-7. Run `gulp release` and have a look at the demo to make sure everything is working correctly.
+### 7. Run `gulp release` and have a look at the demo to make sure everything is working correctly.
```

Comment:
These are nice improvements, great!

On this last point, (7) we can maybe update it to:

```markdown
### 7. Run `npm run build` and have a look at the demo to make sure everything is working correctly.
```

Reply:
Good call!